documentation/akkadoc.shfbproj

```diff
@@ -186,6 +215,30 @@
         The Akka.Persistence.TestKit.Snapshot namespace contains classes that provide base testing functionality used to test
         snapshotting in new persistence extensions.
       </NamespaceSummaryItem>
+      <NamespaceSummaryItem name="Akka.Persistence.Sql.Common" isDocumented="True">
+        The Akka.Persistence.Sql.Common namespace contains common classes used by the various SQL/NoSQL persistence extensions.
```

Comment:
@rogeralsing right. It's used only for SQL stores, as they share many properties.